wiring/src/spark_wiring_logging.cpp

```diff
@@ -629,7 +568,8 @@ Print* spark::DefaultOutputStreamFactory::createStream(const char *type, const J
 void spark::DefaultOutputStreamFactory::destroyStream(Print *stream) {
 #if PLATFORM_ID != 3
     if (stream == &Serial) {
-        Serial.end();
+        // FIXME: Uninitializing the primary USB serial detaches a Gen 3 device from the host
```

Comment:
Given it's a Gen3 issue, should we make it platform conditional?

Reply:
:+1: